versioned_docs/version-2.8/integrations-in-rancher/epinio.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Application Development Engine with Epinio
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/integrations-in-rancher/epinio"/>
+</head>
+
+Epinio lets developers go from application sources to URL in a single step. Epinio is an Application Development Platform. It deploys on Kubernetes and lets application developers and operators work together without conflict. 
+
+## Epinio with Rancher  
+
+Epinio's integration with Rancher lets developers quickly start using it without having to deal with the installation process or the configuration. You can install Epinio from the Apps. Currently the team is working to have Epinio available as a Rancher extension.
+
+## Epinio with Rancher Prime 
+
+On top of the specific support service, Rancher Prime customers of Epinio should expect better integration of Epinio with other Rancher projects such as: 
+
+- Better integration with Rancher authentication.  
+- Integrating Neuvector/Kubewarden with Epinio.
+- Using a custom Chart template with the right annotations to integrate with monitoring for example.
```

Comment:
```suggestion
- Using a custom Chart template with the right annotations to integrate with monitoring for example.
```

Not sure how to revise this to make it snappier..."Custom Helm chart templates for monitoring and other key tools"?